### DISCUSSION TITLE
Is the logs feature working in the Pro version?

### DISCUSSION THREAD

**Discussion by Jono-Moss** [rustdesk/rustdesk#5241] [Q&A] [answered]
Hi, 

I purchased and installed the Pro version yesterday.
It has been working great so far, but an issue I am having is that there appears to be no data in the Logs section.
I have tried to with all the filters and still there seems to be no "Connection" Logs.

The "Console" log section is working but no "Connection" logs are showing.

Thank you

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/issues/21#issuecomment-1637935750

https://github.com/rustdesk/rustdesk-server-pro/issues/13

**Reply by Jono-Moss** [reply]
Thank you, 
I have got port 21114 open and working, I can login to my address book so that is all working.
However, from that client I will connect to a remote computer.
But there is still no Connection logs shown.

**Reply by rustdesk** [reply, marked as answer]
You need to make sure the controlled machine getting port 21114 open and working.

**Reply by Jono-Moss** [reply]
Perfect, thank you very much for your assistance,
Setting the API server in both sides worked.

I would like to make a suggestion that this should perhaps work 
even if only the client initiating the connection has the API server set and not the remote computer.
This will help to log all connections  made from a "Managed Device".

Or if in the future, the Logs could get all connections directly from the "relay server" especially if
the hbbs and hbbr are on the same server / PC.
It would be helpful for creating connection reports in the future.

Thank you again for the awesome software and your speedy help.

**Comment by Craftoncu** [comment]
Hey there, it seems this issue is not resolved yet. 

We also have the 21114 port closed from external. And moved it behind a loadbalancer on external port 443 with certificate

**Reply by open-trade** [reply]
https://github.com/rustdesk/rustdesk/wiki/FAQ#connection-log-or-device-name-on-devices-page-of-web-console-is-empty

**Reply by open-trade** [reply]
https://github.com/rustdesk/rustdesk-server-pro/issues/779
